test/test_dereference.py

```diff
@@ -88,7 +88,6 @@ def test_highly_nested_dereference(self):
 
     def test_dereference_fields(self):
         # Test dereferencing only specific fields.
-
         # Contrived Models that contains more than one ReferenceField at
         # different levels of nesting.
         class MultiReferenceModelEmbed(MongoModel):
```

Comment:
There's a bug in this test. MultiReferenceModelEmbed should inherit from `EmbeddedMongoModel` not `MongoModel`. Perhaps that will fix the failure?

Reply:
Good catch! This one was driving me nuts.